src/tests/test_utils.ts

```diff
@@ -9,116 +9,99 @@ import {
 } from "../types.ts";
 
 export const FAKE_ID = "ABC123";
-export const generatePayload = (
-  id: string,
+export const generateFunctionExecutedPayload = (
+  callback_id: string,
+  enterprise_id?: string,
 ): InvocationPayload<FunctionInvocationBody> => {
   return {
     body: {
       event: {
         type: "function_executed",
-        function: { callback_id: id },
+        function: { callback_id },
         function_execution_id: FAKE_ID,
         bot_access_token: FAKE_ID,
         inputs: {},
       },
-      enterprise_id: FAKE_ID,
+      // Setting to undefined in an effort to model how this property shows up in production: either exists, or it is not defined at all
+      enterprise_id: enterprise_id ? enterprise_id : undefined,
```

Comment:
This is related to my previous comment about how we can write tests specifically crafted to exercise what runtime could would do in a grid vs. not environment. In a grid environment, this property would be present. In a non-grid environment, it would not be.